test-apps/imodel-transformer/src/Main.ts

```diff
@@ -117,31 +117,26 @@ void (async () => {
     }
 
     let requestContext: AuthorizedClientRequestContext | BackendRequestContext;
-    let contextRegistry: ContextRegistryClient | undefined;
+    let iTwinAccessClient: ITwinAccessClient | undefined;
     let sourceDb: IModelDb;
     let targetDb: IModelDb;
     const processChanges = args.sourceStartChangesetIndex || args.sourceStartChangesetId;
 
     if (args.sourceContextId || args.targetContextId) {
       requestContext = await IModelHubUtils.getAuthorizedClientRequestContext();
-      contextRegistry = new ContextRegistryClient();
+      iTwinAccessClient = new ITwinAccessClient();
     } else {
       requestContext = new BackendRequestContext();
     }
 
     if (args.sourceContextId) {
       // source is from iModelHub
       assert(requestContext instanceof AuthorizedClientRequestContext);
-      assert(undefined !== contextRegistry);
+      assert(undefined !== iTwinAccessClient);
       assert(undefined !== args.sourceIModelId);
       const sourceContextId = Guid.normalize(args.sourceContextId);
       const sourceIModelId = Guid.normalize(args.sourceIModelId);
       let sourceEndVersion = IModelVersion.latest();
-      const sourceContext = await contextRegistry.getProject(requestContext, {
-        $filter: `$id+eq+'${sourceContextId}'`,
-      });
-      assert(undefined !== sourceContext);
-      Logger.logInfo(loggerCategory, `sourceContextId=${sourceContextId}, name=${sourceContext.name}`);

```

Comment:
why remove this log message? Is there no replacement to log the iTwinId?

Reply:
The log message was removed because it is the only instance of using the `id` to retrieve an iTwin `name` and is only used in a test application. By removing the log message the interface as a whole can be simplified and easier to maintain.
If you think a logging message is still necessary, I can re-add the log statement with only the `id` but without the `name`.

Reply:
could you re-add it with just the id?

Reply:
Will do!